### PR TITLE
chore: generated CHANGELOG for origin/release/v52.5.0

### PR DESCRIPTION
**Pull request by xyc**
### What does this PR do?
Changelog for v52.5.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+# 52.5.0 - July 14, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- Update test sidebar icon coloring with test results and add test coverage ([PR #3381](https://github.com/forcedotcom/salesforcedx-vscode/pull/3381))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]:
```suggestion
- Update test sidebar icons and add test coverage for easy button replay debugger ([PR #3381](https://github.com/forcedotcom/salesforcedx-vscode/pull/3381))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+#### docs
+
+- Doc Update for Conflict Detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))
+
+- Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
+
+#### salesforcedx-vscode-core
+
+- Remove setting to enable function @W-9311863 ([PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:25]:
```suggestion
- Remove setting to enable function [PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+
+- Conflict detection UI enhancement ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
+
+## Fixed
+
+#### docs
+
+- Doc Update for Conflict Detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))
+
+- Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:21]:
```suggestion
- Disable conflict detection on `SFDX: Retrieve **` commands ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- Update test sidebar icon coloring with test results and add test coverage ([PR #3381](https://github.com/forcedotcom/salesforcedx-vscode/pull/3381))
+
+#### salesforcedx-vscode-core
+
+- Conflict detection update modal ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))
+
+- Conflict detection UI enhancement ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]:
```suggestion
- Enhance hovertip text for conflict detection ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+
+## Added
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- Update test sidebar icon coloring with test results and add test coverage ([PR #3381](https://github.com/forcedotcom/salesforcedx-vscode/pull/3381))
+
+#### salesforcedx-vscode-core
+
+- Conflict detection update modal ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]:
```suggestion
- Update modal text for conflict detection ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,31 @@
...
+
+- Conflict detection UI enhancement ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
+
+## Fixed
+
+#### docs
+
+- Doc Update for Conflict Detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]:
```suggestion
- Update docs for conflict detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,31 @@
...
+
+- Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
+
+#### salesforcedx-vscode-core
+
+- Remove setting to enable function @W-9311863 ([PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))
+
+- Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:27]: Incorporate same change as above

**Comment by AnanyaJha** [comment]
@sbudhirajadoc Let us know what you think!

**Review by sbudhirajadoc — APPROVED** [review]
Looks good!

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 5 above)
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]:
```suggestion
- Update text on Conflict detection modal window ([PR #3394](https://github.com/forcedotcom/salesforcedx-vscode/pull/3394))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 4 above)
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Conflict detection UI enhancement to show last modified date information ([PR #3352](https://github.com/forcedotcom/salesforcedx-vscode/pull/3352))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md]: You can get rid of the GUS workitem Id

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,31 @@
...
+
+- Update docs for conflict detection ([PR #3396](https://github.com/forcedotcom/salesforcedx-vscode/pull/3396))
+
+- Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
+
+#### salesforcedx-vscode-core
+
+- Remove setting to enable function [PR #3409](https://github.com/forcedotcom/salesforcedx-vscode/pull/3409))
+
+- Disable retrieve conflict detection ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```
Comment (xyc) [inline packages/salesforcedx-vscode/CHANGELOG.md:27]:
```suggestion
- Disable conflict detection on `SFDX: Retrieve **` commands ([PR #3382](https://github.com/forcedotcom/salesforcedx-vscode/pull/3382))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 7 above)
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:27]: We can get rid of this. We cherry picked this change to last week's release.

Comment (xyc) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Sounds good, removed

**Review by jag-sfdc — APPROVED** [review]

**Review by AnanyaJha — APPROVED** [review]

**Pull request merged**